src/harness/reference_models/geo/census_tract.py

```diff
@@ -42,3 +43,73 @@ def GetCensusTract(self, fips_code):
+
+  def ExtractData(self, zip_filename=None, fisp_codelist=None):
+    """Extracts the all fisp code matched json file present in specific zip file or all files."""
+    error_occured = False
+
+    # Filter the zip filename based on specified file name if any endswith .zip
+    census_tracts_file_list = [os.path.join(self._census_tract_dir, f)
+                               for f in os.listdir(self._census_tract_dir)
+                               if all((True if not zip_filename else
+                                       f.startswith(zip_file_name),
+                                       f.endswith('.zip')))]
+    if not census_tracts_file_list:
+      print 'Please specify the correct zip file name or clone SAS-Data directory before' \
+            'execute this script'
+      error_occured = True
+
+    # Filter the files matches with fisp_code in all downloaded zipped file and matched files
+    # will be extracted.
+    for file_name in census_tracts_file_list:
+        # Check if already extracted
+        zf = zipfile.ZipFile(file_name, 'r')
+
+        #Only fisp_code matched json files are fetched from specified zip file if it is specified.
+        #if fisp_codes are not specified then all the json files from either specific zipfile
+        #or all zipped file that containing the json files will be returned.
+        fisp_code_matched_files = [datfile
+                                   for datfile in zf.infolist()
+                                   if all((datfile.filename.endswith('.json'),
+                                           True if not fisp_code_list else datfile.filename in
+                                           fisp_codelist))]
+        if fisp_code_matched_files:
+          break
+
+    if not fisp_code_matched_files:
+      print 'No fisp code matched files present in zip file. Please verify the inputs'
+      error_occured = True
+
+    for datfile in fisp_code_matched_files:
+      try:
+        zf.extract(datfile, self._census_tract_dir)
+        print '%s is extracted successfully in %s' % (datfile.filename,
+                                                      self._census_tract_dir)
+      except Exception as err:
+        error_occured = True
+        print "Exception:%s" % err.message
+        print 'Cannot extract ' + datfile.filename + ' from ' + os.path.dirname(
+            datfile.filename)
+
+    # Check is there any exception while extracting the files.
+    if not error_occured:
+       print 'SUCCESS: All requested files which are matched with fisp code ' \
+             'are extracted successfully'
+    else:
+       print 'FAILURE: There is an exception while extracting some of the files \n or '  \
+             'No matched files are returned.\n ' \
+             'Please specify the correct zip file name or clone SAS-Data directory before' \
+             'execute this script'
+
+# Set the Base data directory
```

Comment:
Please do not add code on the library that will be run when importing this driver module.

All scripts which generated the census database should be in src/data/, and of course they can use this library if required,